app/src/main/res/layout/fragment_registration.xml

```diff
@@ -13,7 +13,7 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:hint="@string/account_number"
-            android:inputType="textEmailAddress"/>
+            android:inputType="number"/>
```

Comment:
@Rish02Sharma account number should be text type. It can consist a-z and 0-9 etc.